apps/www/src/registry/default/example/playground-demo.tsx

```diff
@@ -142,7 +142,12 @@ export const usePlaygroundPlugins = ({
           }),
           createImagePlugin({ enabled: !!enabled.img }),
           createMediaEmbedPlugin({ enabled: !!enabled.media_embed }),
-          createMentionPlugin({ enabled: !!enabled.mention }),
+          createMentionPlugin({
+            enabled: !!enabled.mention,
+            options: {
+              triggerPreviousCharPattern: /[\s()]/,
```

Comment:
Just to make sure I'm reading this correctly, this matches whitespace and literal `(` and `)` characters, right? At first glance, I didn't spot the square brackets and thought it was an empty capture group.

Reply:
Yep, right.

I tried with escaping but was unnecessary: received an alert with `/[\s\(\)]/ can be optimized to /[\s()]/.`. Same alert triggered when using `/[\s)(]/`

I will change this example to include quotes. Say `"` or `'`

Reply:
Perfect. Thanks for the clarification!

In my case, the regex is clearer even when having that "empty group". Would be something like:

```
/^$|^[\s"'(),.[\]`{}]$/
```

(ordered by eslint [unicorn/better-regex](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/v47.0.0/docs/rules/better-regex.md)).

This means basically match the beginning of line, any parenthesis, brackets, quotes, etc.